sigstore/newtypes.py

```diff
@@ -0,0 +1,6 @@
+from typing import NewType
+
+Hexstr = NewType('hexstr', str)
+B64str = NewType('b64str', str)
+Pemcert = NewType("pemcert", str)
+KeyID = NewType('keyID', bytes)
```

Comment:
The `NewType` API expects the LHS of the expression and the first argument to match, so these should be:

```suggestion
hexstr = NewType('hexstr', str)
b64str = NewType('b64str', str)
pemcert = NewType("pemcert", str)
keyid = NewType('keyid', bytes)
```

(I made these all lowercase because they're all `NewType`s of primitive types, but it might make sense to have them be `PEMCert` etc. instead. But we can think more about that in a later review.)